src/applications/personalization/preferences/tests/e2e/emptyPreferences.e2e.spec.js

```diff
@@ -13,60 +13,6 @@ function emptyPreferences(browser, token) {
   initChoicesGetMock(token);
   initHealthPostMock(token);
 
-  // init announcements
-  browser.execute(() => {
-    window.localStorage.setItem(
-      'DISMISSED_ANNOUNCEMENTS',
-      '["welcome-to-new-va"]',
-    );
-  });
-
-  // verify FTUX Find VA benefits modal
-  browser
-    .url(`${E2eHelpers.baseUrl}/my-va`)
-    .waitForElementVisible('.va-modal-body', Timeouts.slow);
-  browser.assert.containsText('.va-modal-body', 'Find VA benefits');
-
-  // accept find benefits modal
-  browser
-    .click('.va-modal-body .usa-button')
-    // land on correct page
-    .waitForElementVisible('#dashboard-title', Timeouts.slow);
-  browser.assert.containsText('#dashboard-title', 'Find VA benefits');
-
-  // init announcements
-  browser.execute(() => {
-    // window.localStorage.clear();
-    window.localStorage.setItem(
-      'DISMISSED_ANNOUNCEMENTS',
-      '["welcome-to-new-va"]',
-    );
-  });
-
-  // verify modal
-  browser
-    .url(`${E2eHelpers.baseUrl}/my-va`)
-    .waitForElementVisible('.va-modal-body', Timeouts.slow);
-  browser.assert.containsText('.va-modal-body', 'Find VA benefits');
-
-  // decline find benefits modal
-  browser
-    .click('.va-modal-body .usa-button-secondary')
-    // land on correct page
-    .waitForElementVisible('#dashboard-title', Timeouts.slow);
-  browser.assert.containsText('#dashboard-title', 'My VA');
-
-  // init announcements
-  browser.execute(() => {
-    window.localStorage.setItem('DISMISSED_ANNOUNCEMENTS', '*');
-  });
-
-  // ensure that modal does not appear if it has been dismissed
-  browser
-    .url(`${E2eHelpers.baseUrl}/my-va`)
-    .waitForElementVisible('body', Timeouts.slow);
-  browser.assert.elementNotPresent('.va-modal-body');
-
```

Comment:
This e2e test isn't really the place to test site-wide announcements, unless we're going to keep the `window.localStorage.setItem(` calls up-to-date with how the announcement config changes. I don't think we're going to do that.

Reply:
Made a note for us to create announcement e2e tests here: https://dsva.slack.com/archives/C52CL1PKQ/p1587412119244800